src/main/java/de/tum/in/www1/artemis/web/rest/TutorParticipationResource.java

```diff
@@ -90,9 +90,10 @@ public TutorParticipationResource(TutorParticipationService tutorParticipationSe
     }
 
     /**
-     * POST /exercises/:exerciseId/assess-example-submission: Add an example submission to the tutor participation of the given exercise. If it is just for review (not used for tutorial),
-     * the method just records that the tutor has read it. If it is a tutorial, the method checks if the assessment given by the tutor is close enough to the instructor one. If
-     * yes, then it returns the participation, if not, it returns an error
+     * POST /exercises/:exerciseId/assess-example-submission: Add an example submission to the tutor participation of the given exercise.
+     * If it is just for review (not used for tutorial), the method just records that the tutor has read it.
+     * If it is a tutorial, the method checks if the assessment given by the tutor is matches the instructor one.
```

Comment:
"if the assessment given by the tutor _(remove the 'is' here)_ matches the instructor one."